README.md

```diff
@@ -207,6 +213,12 @@ If you want to contribute to Auto-PyTorch, clone the repository and checkout our
 $ git checkout development
 ```
 
+### Community Contribution
+1. Fork the Repository into your account
+2. create a `feature` or `bug` sub-branch e.g. `git checkout -b feature/name_of_feature`
```

Comment:
As stated in line 210, we would like the users to check out new branches from the development branch. I suggest that you unify `## Contributing` and `### Community Contribution`.